test/TritonGPU/swp-warning.mlir

```diff
@@ -0,0 +1,40 @@
+// RUN: triton-opt %s -split-input-file -tritongpu-pipeline=num-stages=3 -dump-swp-failure | FileCheck %s
+
+// CHECK-LABEL: @dont_pipeline_128x1
+// CHECK-NOT: local_load{{.*}}128x1
```

Comment:
this doesn't seem to test the code added?

Reply:
Tests for warnings are added here: python/test/unit/test_perf_warning.py. Can you see if you can add a case there?